dvc/remote/local.py

```diff
@@ -743,3 +744,6 @@ def _log_missing_caches(self, checksum_info_dict):
                 "nor on remote. Missing cache files: {}".format(missing_desc)
             )
             logger.warning(msg)
+
+    def unportect(self, target):
+        unprotect(target)
```

Comment:
This is backwards. Move repo.unprotect logic to RemoteLOCAL.unprotect and use it.

Reply:
Done

Reply:
Please move logic from `repo.unprotect()` to `RemoteLOCAL.unprotect()` and use `RemoteLOCAL.unprotect` in `repo.unprotect()`.